coc-settings.json

```diff
@@ -0,0 +1,27 @@
+{
+  "coc.preferences.formatOnSaveFiletypes": [
```

Comment:
This doesn't seem to work. I have autocommands setup that cause the formatting to be started.